src/link-manager.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @fileoverview Link manager for controlling the link query.
+ */
+
+/** @const */
+genotet.linkManager = {};
+
+/**
+ * Array that stores the preset view names of all the views.
+ */
+/** @type {!Array<string>} */
+genotet.linkManager.presetStatus = [];
```

Comment:
How will this be used later? Linked queries a set of action->response definitions. For example, {clicking a node in the network}->{loading corresponding binding data}.

The link manager should store such (action, response) pairs, rather than a list of preset views. I don't think a list of preset views would be useful later.